src/pages/Categories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+// eslint-disable-next-line
+import { checkStatus } from "../redux/categories/categories";
```

Comment:
- [ ] As requested by @Gambit142 kindly remove the Eslint-disable statement from this file. The Eslint helps us write clean, maintainable, and scalable codes. Using Eslint-disable statements defeats this purpose.
- [ ] Please 🙏🏽 correct the path, it should be "../redux/Categories/categories" to avoid compilation errors.